codeListAPI/get_editions_test.go

```diff
@@ -0,0 +1,69 @@
+package codeListAPI
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/gavv/httpexpect"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestSuccessfulRetrievalOfEditions(t *testing.T) {
+
+	if err := ds.DropCodelists(); err != nil {
+		t.Error(err)
+		t.Fail()
+	}
+
+	if err := ds.SetupCodelistEditions(); err != nil {
+		t.Error(err)
+		t.Fail()
+	}
+
+	codelistAPI := httpexpect.New(t, cfg.CodeListAPIURL)
+
+	Convey("Given two editions of the same codelist exists", t, func() {
+		Convey("When a request is made for the editions", func() {
+			Convey("Then the codelists are returned to the user", func() {
```

Comment:
the editions are returned to the user, rather than codelists, as there's only one list at this point?